orchestra/management/commands/loadworkflow.py

```diff
@@ -0,0 +1,35 @@
+from django.core.management.base import BaseCommand, CommandError
+from orchestra.workflow.exceptions import WorkflowError
+from orchestra.workflow.load import load_workflow
+
+import os
+
+
+class Command(BaseCommand):
+    help = 'Loads a workflow version into the database'
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            'workflow_directory',
+            help='The code directory containing the workflow.')
+        parser.add_argument(
+            'version_slug', nargs='?', default=None,
+            help=('(Optional) The version to load. If omitted, all available '
+                  'versions are loaded.'))
+        parser.add_argument(
+            '--force', action='store_true', default=False,
+            help=('If a workflow version already exists, update the existing '
+                  'version instead of exiting.'))
+
+    def handle(self, *args, **options):
+        workflow_dir = os.path.abspath(options['workflow_directory'])
```

Comment:
it'd be nice if this called code in `orchestra` for any business logic, so the script was just the wrapper around that code